kit/ForKit.cpp

```diff
@@ -366,6 +361,8 @@ static void cleanupChildren()
     }
 }
 
+static bool lokitProcessIsRunning = false;
```

Comment:
Please add comments so it is clear what the various variables/functions do. It's not clear what this is supposed to represent and how it's supposed to work.